onadata/libs/serializers/organization_serializer.py

```diff
@@ -85,6 +85,16 @@ def __init__(self, *args, **kwargs):
                 for field in getattr(self.Meta, "owner_only_fields"):
                     self.fields.pop(field)
 
+    def to_representation(self, instance):
+        """Override `to_representation`."""
+        data = super().to_representation(instance)
+
+        if not self._is_admin_or_manager(self.context["request"].user, instance):
+            data.pop("active_kms_key")
+            data.pop("inactive_kms_keys")
```

Comment:
Can we push this to the serializer methods for the relevant fields? Or perhaps have a different serializer for managers that includes these fields so that we don't end up with pop()?